src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
+
+const clickPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject('Promise was rejected!');
+  }, 3000);
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+    clearTimeout(timeoutId);
+  });
+});
+
+const messageDiv = document.createElement('div');
+
+document.body.append(messageDiv);
+
+clickPromise
+  .then((message) => {
+    messageDiv.classList.add('message');
```

Comment:
The 'message' class is added to the div in both the `then` and `catch` blocks. To avoid repetition, you could add this class right after the div is created.